skimage/measure/tests/test_simple_metrics.py

```diff
@@ -43,10 +43,10 @@ def test_PSNR_float():
 def test_PSNR_dynamic_range_and_data_range():
     # Tests deprecation of "dynamic_range" in favor of "data_range"
     out1 = compare_psnr(cam/255., cam_noisy/255., data_range=1)
-    with expected_warnings(
+    with expected_warnings([
             '`dynamic_range` has been deprecated in favor of '
             '`data_range`. The `dynamic_range` keyword argument '
-            'will be removed in v0.14'):
+            'will be removed in v0.14']):
```

Comment:
Erm, why is this still here???? But anyway, I'll leave a fix to a later PR. @soupault?

Reply:
@jni My bad. :) I'll take care of this.